### DISCUSSION TITLE
How to add projectId inside walleConnectWallet?

### DISCUSSION THREAD

**Discussion by deantek** [rainbow-me/rainbowkit#1125] [Q&A]
I ran into a problem that transactions do not work through wallet connect, I looked at the WC documentation, they write there that I need to use projectId, but I can’t do it, typescript gives an error

````
const connectors = connectorsForWallets([
  {
    groupName: 'Popular',
    wallets: [metaMaskWallet({ chains }), walletConnectWallet({ chains, projectId ??? })],
  },
  {
    groupName: 'Other',
    wallets: [ledgerWallet({ chains }), injectedWallet({ chains })],
  },
])
````

in the Wallet connect documentation they connect like this, but how do I get it to work with RainbowKit?

````
new WalletConnectConnector({
      chains,
      options: {
        projectId: '',
      },
    }),
````

**Comment by myr45500** [comment]
Try writing  // @ts-ignore above the line where it gives an error, like so

  // @ts-ignore
   wallets: [metaMaskWallet({ chains }), walletConnectWallet({ chains, projectId ??? })],

If it still shows an error, can you hover over the error, and provide us what kind of error it saying

**Comment by BShola1993** [comment]
i want to add projectId  to my project. from rainbow kit. i am working with nextjs/react.  Hree is my snippet ```"use client"
import {getDefaultConfig} from "@rainbow-me/rainbowkit";
import {anvil,zksync, sepolia,op mainnet} from "wagmi/chains"
export default getDefaultConfig({
    appName: "Tsender",
    projectId:null,
    chains:[anvil,zksync,op mainnet],
    ssr:false
})```

**Reply by DanielSinclair** [reply]
@BShola1993 You can get a projectId from here: https://cloud.reown.com/sign-in

**Reply by BShola1993** [reply]
Oh thanks Sinclair
